test/regress/regress0/parser/witness.cvc

```diff
@@ -0,0 +1,10 @@
+% EXPECT: sat
```

Comment:
I believe this benchmark should be deleted (we do not support WITNESS in parsers).